lib/method_execution/array_destructer.rb

```diff
@@ -0,0 +1,13 @@
+module ArrayDestructer
```

Comment:
Why two different modules with the same names and methods just different locations?